test/test_torch.py

```diff
@@ -1391,7 +1398,12 @@ def _test_broadcast(self, cast):
                 small2 = cast(torch.randn(*dims_small2).float())
                 small2_expanded = small2.expand(*dims_full)
 
-            if hasattr(large_expanded, fn):
+            if small.is_cuda and fn in ['map', 'map2']:
+                # map and map2 are not implementd on CUDA tensors
```

Comment:
is it easy to throw something in here that will fail if we implement these?  Like a assertRaises or something?  Otherwise people will invent an entirely new test when they implement this instead of just getting rid of this.

Reply:
Adding a new test for these functions seems like a good outcome.